### DISCUSSION TITLE
How to set weights between target position and target rotation in mpc?

### DISCUSSION THREAD

**Discussion by Spphire** [NVlabs/curobo#69] [Software Q&A] [answered]
I found that rotation is not that accurate as position, I want to add more weight of target rotation in cost

**Comment by balakumar-s** [comment]
Is this for IK, MPC or motion_gen?

**Reply by Spphire** [reply]
MPC

**Reply by balakumar-s** [reply, marked as answer]
Increasing the first value of this list will increase the weight for orientation: https://github.com/NVlabs/curobo/blob/1458f79a316855efde6ad314d5cffb20f10aecf0/src/curobo/content/configs/task/particle_mpc.yml#L34
